sample/simple/settings.gradle.kts

```diff
@@ -13,5 +13,5 @@ plugins {
 rootProject.name = "simple-sample"
 
 configure<com.freeletics.gradle.plugin.SettingsExtension> {
-    includeKhonshu("../..", true)
+    includeKhonshu("../..")
```

Comment:
When `true` is passed the `includedBuild` will completely replace `navigation-compose` dependencies with `navigation-experimental` while using `false` (the default) will replace each of them with the matching local module.

Reply:
With this change it probably makes sense to remove the boolean option in FGP